ocfweb/reservations/mail_templates/user_notification.jinja

```diff
@@ -0,0 +1,8 @@
+Hello {{request.real_name}},
+
+The OCF has received your request to reserve the lab on {{request.date}} from {{request.starttime}} to {{request.endtime}} and is pending staff approval.
+Your request is currently being processed. Should we encounter any difficulty or require any additional information, we will send you an email.
+Once we've made a decision, we'll send you an email to let you know.
+
+If, for whatever reason, you need to contact us about this reservation request, please don't hesitate to send an email to awelty@ocf.berkeley.edu.
```

Comment:
awelty?

Reply:
I used personal emails to test so as not to spam. These should all be fixed now.